app/facets/sandbox/sandbox.coffee

```diff
@@ -29,7 +29,12 @@ class Sandbox
 			]
 
 		@match = new nx.Cell
-			'<-': [@remote.match]
+			'<-': [
+				@remote.match
+				(match) ->
+					match.time_stamp = do Date.now
```

Comment:
Arguably this should be done on Sandbox as it is more accurate (without the roundtrip etc.)